src/main/resources/com/google/api/codegen/php/method_sample.snip

```diff
@@ -32,37 +33,39 @@
 @end
 
 @snippet sampleCode(apiMethod)
-    @if apiMethod.isLongRunningOperation
-        {@longRunningMethodSampleCode(apiMethod)}
-    @else
-      @switch apiMethod.grpcStreamingType
-      @case "NonStreaming"
-          @switch apiMethod.type
-          @case "OptionalArrayMethod"
-              {@methodSampleCode(apiMethod)}
-          @case "PagedOptionalArrayMethod"
-              {@pagedMethodSampleCode(apiMethod)}
-          @default
-              $unhandledCase: {@apiMethod.type}$
-          @end
-      @case "ClientStreaming"
-          {@clientStreamingMethodSampleCode(apiMethod)}
-      @case "ServerStreaming"
-          {@serverStreamingMethodSampleCode(apiMethod)}
-      @case "BidiStreaming"
-          {@bidiStreamingMethodSampleCode(apiMethod)}
-      @default
-          $unhandledCase: {@apiMethod.grpcStreamingType}$
-      @end
+    {@incodeSamples(apiMethod)}
+@end
 
+@snippet incodeSamples(apiMethod)
+    @join sample : apiMethod.samples on BREAK.add(BREAK).add(BREAK).add("// Alternatively:").add(BREAK)
+        {@incodeSampleCall(apiMethod, sample)}
```

Comment:
I think the `{@initCode...}` from `decorateSampleCode` should be moved here, right above {@incodeSampleCall....}

Also (IMPORTANT), that `{@initCode ... }` line should not be referencing `apiMethod.initCode`, but rather `sample.initCode`. (We're deprecating use of method-level `initCode` in favor of sample-level `initCode`. The infrastructure is in place so that if only the former is configured in the YAML, it will be available as the latter by the time we create view models.)

Now, for methods with multiple in-code samples, this would repeat the setup code. To avoid that for situations where the setup code is the same, you could implement something like `py/method_samples.snip:12:incodeSamples()`